_drafts/2017-03-20-osd-needs-better-collaboration-tools.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  Open source design needs better collaboration tools
+date:   2017-03-09
+categories:
+  - open-source
+  - communities
+  - collaboration
+  - tools
+  - design
+---
+
+Despite the rising awareness and acceptance of UX design, in particular on the web, it has failed to gain much traction in open source software. If for argument's sake, we take UX design to have started in 1995 when [Don Norman started work for Apple as a "user experience architect"](http://blog.invisionapp.com/a-brief-history-of-user-experience/), even though it has a longer history, then the fact that design has failed to make much impact in the open source world for over 20 years suggests that there are structural and systemic barriers that make design and open source development as compatible to each other as oil and water.
+
+In this post, I will argue the need for better communication and collaboration platforms by looking at some of the fundamental characteristics of open source design and development and propose some starting points.
+
+The goal of OSD is to improve the user experience of open source software. Because of the nature of open source software development, to achieve this goal, certain critical infrastructure is necessary. I am talking about a way for designers to share and build upon ideas, interact, reach consensus and collaborate with other designers and all the other stakeholders involved in the creation of software.
```

Comment:
"Because of the nature of open source software development certain critical infrastructure is still needed to achieve this goal." <- rewrite. The commas are odd rhythm.

I also wonder what you mean by "the nature of open source software".